sigs.yml

```
@@ -0,0 +1,502 @@
+    notes:
+      type: gDoc
+      value: 1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE
+    chat:
+      - type: slack
+        name: '#otel-messaging'
+        id: C02Q4AAHDSA
+    invites: calendar-semconv
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Semantic Conventions: System Metrics'
+    meeting: Every Thursday at 08:30 PT
+    notes:
+      type: gDoc
+      value: 1p5TH57t43XpxA48onLzX4PIr3g6ydYKCtR_AUlsCnQk
+    chat:
+      - type: slack
+        name: '#otel-system-metrics-wg'
+        id: C05CTFE9U4A
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Josh Suereth
+      github: jsuereth
+    gcLiaison: []
+  - name: 'Semantic Conventions: LLM'
+    meeting: Every Wednesday alternating between 08:00 and 16:00 PT
+    notes:
+      type: gDoc
+      value: 1EKIeDgBGXQPGehUigIRLwAUpRGa7-1kXB736EaYuJ2M
+    chat:
+      - type: slack
+        name: '#otel-llm-semconv-wg'
+        id: C06KR7ARS3X
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Liudmila Molkova
+      github: lmolkova
+    gcLiaison: 
+    - name: Alolita Sharma
+      github: alolita
+- name: Implementation SIGs
+  sigs:
+  - name: Maintainer Meeting
+    meeting: Every Monday at 09:00PT
+    notes:
+      type: gDoc
+      value: 1_X2CTiDXmxk2JozBNwQ2uP8FfV1rxx9ISE5puHu0yIk
+    chat:
+      - type: slack
+        name: '#otel-maintainers'
+        id: C01NJ7V1KRC
+    invites: calendar-maintainer-meeting
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Android: SDK + Automatic Instrumentation'
+    meeting: Every Tuesday at 08:00PT
+    notes:
+      type: gDoc
+      value: 1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY
+    chat:
+      - type: slack
+        name: '#otel-android'
+        id: C05J0T9K27Q
+    invites: calendar-android
+    tcSponsors: []
+    gcLiaison: []
+  - name: Collector
+    meeting: Every Wednesday at 09:00 PT plus monthly on first Wednesday at 00:00 PT
+      and on third Wednesday at 16:00 PT
+    notes:
+      type: gDoc
+      value: 1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE
+    chat:
+      - type: slack
+        name: '#otel-collector'
+        id: C01N6P7KR6W
+    invites: calendar-collector
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Juraci Paixão Kröhling'
+      github: jpkrohling
+  - name: 'C++: SDK'
+    meeting: Every week alternating between Monday at 13:00 PT and Wednesday at 9:00
+      PT
+    notes:
+      type: gDoc
+      value: 1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw
+    chat:
+      - type: slack
+        name: '#otel-cpp'
+        id: C01N3AT62SJ
+    invites: calendar-cpp
+    tcSponsors: []
+    gcLiaison: []
+  - name: '.NET: Automatic Instrumentation'
+    meeting: Every Wednesday at 9:00 PT
+    notes:
+      type: gDoc
+      value: 1dYdwRQVE3zu0vlp_lqGctNm0dCQUkDo2LfScUJzpuT8
+    chat:
+      - type: slack
+        name: '#otel-dotnet-auto-instr'
+        id: C01NR1YLSE7
+    invites: calendar-dotnet
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG